src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -224,6 +225,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     options_list=['--object-id', '-i'],
     help='The unique ID of the Azure AD administrator.')
 
+read_scale_param_type = CLIArgumentType(
+    options_list=['--read-scale'],
+    help='If enabled, connections that have application intent set to readonly '
+    'in their connection string may be routed to a readonly secondary replica. '
+    'This property is only settable for Premium and Business Critical databases.',
+    arg_type=get_enum_type(DatabaseReadScale))
```

Comment:
What are DatabaseReadScale values or type? Boolean? 
In addition, can user disable this property?

Reply:
I took a look at this and am concerned this will introduce some confusion and inconsistency.  The REST API and SDK expects "Enabled" or "Disabled" and Powershell client also expects "Enabled" and "Disabled" parameter values.  If CLI only starts using "True" and "False" it will not be a consistent experience.  I agree this should've been done as boolean from the beginning, but this API has been around now for 3 years and I don't want to make CLI only to be different.  If we want to make this change, I think API and all clients should be changed together.

Reply:
You can initialize your SDK model  to "Enabled" when the parameter is true and "Disabled" when false.  It will be user friendly.

Reply:
We are not changing the SDK as it has already been released for 3 years and we have not had any issue with usage.  It's well documented and easy to use "Enabled" and "Disabled" values.